util/__init__.py

```diff
@@ -0,0 +1,4 @@
+import re
+
+def key_to_ckey(key):
+	return re.sub('[^A-Za-z0-9]+', '', player)
```

Comment:
Using the wrong variable
```suggestion
	return re.sub('[^A-Za-z0-9]+', '', key)

```